/usr/bin/ld: cannot find -lcudart_static

Hello,
I get a linker error at the end of the compilation on Arch Linux.
Any idea's what i'm doing wrong?
Thanks!
*********************
$ export CC=/usr/bin/clang

$ export CXX=/usr/bin/clang++

$ cmake . -DCUDA_COMPILER=clang -DMICROHTTPD_REQUIRED=OFF
-- The C compiler identification is Clang 4.0.0
-- The CXX compiler identification is Clang 4.0.0
-- Check for working C compiler: /usr/bin/clang
-- Check for working C compiler: /usr/bin/clang -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/clang++
-- Check for working CXX compiler: /usr/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found CUDA: /opt/cuda (found suitable version "8.0", minimum required is "7.5") 
-- Found OpenSSL: /usr/lib/libcrypto.so (found version "1.1.0f") 
-- Configuring done
-- Generating done
-- Build files have been written to: /home/source/xmr-stak-nvidia
$


$ make
Scanning dependencies of target xmr-stak-nvidiaCrypto
[  4%] Building C object CMakeFiles/xmr-stak-nvidiaCrypto.dir/crypto/c_blake256.c.o
[  9%] Building C object CMakeFiles/xmr-stak-nvidiaCrypto.dir/crypto/c_groestl.c.o
[ 14%] Building C object CMakeFiles/xmr-stak-nvidiaCrypto.dir/crypto/c_jh.c.o
[ 19%] Building C object CMakeFiles/xmr-stak-nvidiaCrypto.dir/crypto/c_keccak.c.o
[ 23%] Building C object CMakeFiles/xmr-stak-nvidiaCrypto.dir/crypto/c_skein.c.o
[ 28%] Building C object CMakeFiles/xmr-stak-nvidiaCrypto.dir/crypto/soft_aes.c.o
[ 33%] Linking C static library libxmr-stak-nvidiaCrypto.a
[ 33%] Built target xmr-stak-nvidiaCrypto
Scanning dependencies of target xmr-stak-nvidiaCuda
[ 38%] Building CXX object CMakeFiles/xmr-stak-nvidiaCuda.dir/nvcc_code/cuda_core.cu.o
[ 42%] Building CXX object CMakeFiles/xmr-stak-nvidiaCuda.dir/nvcc_code/cuda_extra.cu.o
[ 47%] Linking CXX static library libxmr-stak-nvidiaCuda.a
[ 47%] Built target xmr-stak-nvidiaCuda
Scanning dependencies of target xmr-stak-nvidia
[ 52%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/cli-miner.cpp.o
[ 57%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/console.cpp.o
[ 61%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/executor.cpp.o
[ 66%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/httpd.cpp.o
[ 71%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/jconf.cpp.o
[ 76%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/jpsock.cpp.o
[ 80%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/minethd.cpp.o
[ 85%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/socket.cpp.o
[ 90%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/webdesign.cpp.o
[ 95%] Building CXX object CMakeFiles/xmr-stak-nvidia.dir/crypto/cryptonight_common.cpp.o
[100%] Linking CXX executable bin/xmr-stak-nvidia
/usr/bin/ld: cannot find -lcudart_static
clang-4.0: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [CMakeFiles/xmr-stak-nvidia.dir/build.make:340: bin/xmr-stak-nvidia] Error 1
make[1]: *** [CMakeFiles/Makefile2:143: CMakeFiles/xmr-stak-nvidia.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
$

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -87,7 +87,6 @@ option(CUDA_SHOW_REGISTER "Show registers used for each kernel and compute archi
 option(CUDA_KEEP_FILES "Keep all intermediate files that are generated during internal compilation steps" OFF)
 
 if("${CUDA_COMPILER}" STREQUAL "clang")
-    set(LIBS ${LIBS} cudart_static)
     set(CLANG_BUILD_FLAGS "-O3 -x cuda --cuda-path=${CUDA_TOOLKIT_ROOT_DIR}")
     # activation usage of FMA
     set(CLANG_BUILD_FLAGS "${CLANG_BUILD_FLAGS} -ffp-contract=fast")
@@ -209,6 +208,7 @@ if("${CUDA_COMPILER}" STREQUAL "clang")
     set_target_properties(xmr-stak-nvidiaCuda PROPERTIES COMPILE_FLAGS ${CLANG_BUILD_FLAGS})
     set_target_properties(xmr-stak-nvidiaCuda PROPERTIES LINKER_LANGUAGE CXX)
     set_source_files_properties(${CUDASRCFILES} PROPERTIES LANGUAGE CXX)
+    target_link_libraries(xmr-stak-nvidiaCuda PUBLIC ${LIBS})
 else()
     # build device code with nvcc
     cuda_add_library(xmr-stak-nvidiaCuda

